src/main/java/org/kumoricon/registration/order/ReprintController.java

```diff
@@ -32,8 +32,8 @@ public ReprintController(BadgePrintService badgePrintService, AttendeeRepository
         this.attendeeHistoryRepository = attendeeHistoryRepository;
     }
 
-    @RequestMapping(value = "/orders/{orderId}/attendees/{attendeeId}/reprint", method = RequestMethod.POST)
-    @PreAuthorize("hasAuthority('reprint_badge')")
+    @RequestMapping(value = "/orders/{orderId}/attendees/{attendeeId}/reprint")
+    @PreAuthorize("hasAuthority('reprint_badge') || hasAuthority('reprint_badge_with_override')")
```

Comment:
main issue with this is if a user manually enters /reprint then they can reprint the badge of the user, if that's an issue i can recreate this and probably move a reprint method to OrderAttendeeEditController